lib/models/device_folder.dart

```diff
@@ -11,3 +11,25 @@ class DeviceFolder {
     this.thumbnail,
   );
 }
+
+class DevicePathCollection {
```

Comment:
Is `DeviceFolder` a better name? We could rename the above one to something else (not sure where all it's used anymore)